### ISSUE TITLE
Web-bugs flashes when loading.

### ISSUE THREAD

**Issue by MarjaE2** [webcompat/webcompat.com#3647]
See https://github.com/webcompat/web-bugs/issues/93435

URL: https://github.com/webcompat/web-bugs

Browser/Version and Operating System: Tested with multiple browsers, starting with Firefox 94. Most flash and jump as they load the page.

Expected Behavior: No flashing.

Actual Behavior: Flashing and jumping as the page loads.

Migraine warnings for flashing in all screenshots.

In Firefox 94 with various accessibility fixes:

https://user-images.githubusercontent.com/12875862/143024293-d14ced9b-34cc-44cd-b9b0-9ffbfe05e052.mov

In Safari:

https://user-images.githubusercontent.com/12875862/143024465-e113bb8e-796c-47de-9b60-9c83973f9bd0.mov

In Firefox Nightly from November 17th with default settings:

https://user-images.githubusercontent.com/12875862/143024602-2aa97a8d-7893-4a1a-a3cf-34ebdb3adc0b.mov

**Comment by softvision-raul-bucata** [comment]
@MarjaE2 Does the issue occur on your side when the refresh of the page is triggered?

**Comment by MarjaE2** [comment]
Yes. It's much easier for me to test w/ refresh than with the initial load.
